packages/braintree-integration/src/BraintreeAchPaymentMethod.tsx

```diff
@@ -55,8 +73,21 @@ const BraintreeAchPaymentMethod: FunctionComponent<PaymentMethodProps> = ({
             }
         };
 
+        const initializeInstruments = async () => {
```

Comment:
[optional]
I think we can use more meaningful naming for the function. What do you think?

```suggestion
        const loadInstrumentsOrThrow = async () => {
```